library/modules/bigip_gtm_dns_listener.py

```diff
@@ -0,0 +1,890 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2020, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_gtm_dns_listener
+short_description: configures BIG-IP DNS system to answer TCP or UDP DNS requests
+description:
+  - Defines one or more listener objects to control what protocols are available for the BIG-IP DNS system to process DNS requests.
+  - BIG-IP DNS listeners allows to use TCP and UDP protocols.
+notes:
+  - Requires BIG-IP version 12.0.0 or greater
+version_added: 2.8
```

Comment:
wrong version

Reply:
I have updated it to 
```version_added: "f5_modules 1.4"```
 This comment is addressed now